src/front/glsl/ast.rs

```diff
@@ -36,6 +39,10 @@ pub struct FunctionDeclaration {
     pub defined: bool,
     /// Wheter or not this function returns void (nothing)
     pub void: bool,
+    /// [`BitSet`](BitSet) where each bit indicates if the argument in the
+    /// corresponding position should be treated as a depth image instead of a
+    /// regular image
+    pub depth_set: BitSet,
```

Comment:
would it make sense to just make it a boolean field in some `struct Parameter`? there is already `parameters: Vec<>` here to store this